### PR TITLE
x11-libs/goocanvas: bump to 3.0.0 and EAPI 8

### PR DESCRIPTION
**Pull request by scardracs**
Closes: https://bugs.gentoo.org/783018

Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	>=x11-libs/gtk+-3.0.0:3
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+DEPEND="
+	${RDEPEND}
+	${PYTHON_DEPS}
+	>=sys-devel/gettext-0.19.4
+	dev-util/glib-utils
+	>=dev-util/gtk-doc-am-1.16
+	virtual/pkgconfig
```
Comment (thesamesam) [inline x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild:30]: All 4 of these are BDEPEND, and I think ${PYTHON_DEPS} should be too :)

File: x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+"
+
+src_prepare() {
+	# Do not build demos
+	sed -e 's/^\(SUBDIRS =.*\)demo\(.*\)$/\1\2/' \
+		-i Makefile.am Makefile.in || die "sed failed"
+
+	# Python bindings are built/installed manually, but not at all anymore (py2).
+	sed -e "/SUBDIRS = python/d" -i bindings/Makefile.am bindings/Makefile.in
```
Comment (thesamesam) [inline x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild:39]: Missing `|| die`.

File: x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild (hunk 1 above)
Comment (scardracs) [reply x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild:30]: Done :)

File: x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild (hunk 2 above)
Comment (scardracs) [reply x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild]: Already done, I've noticed it just before your comment ':)

**Comment by scardracs** [comment]
Fecthed and repushed, hopefully this will finally fix the error with badvpn

File: x11-libs/goocanvas/goocanvas-3.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+IUSE="examples +introspection"
+
+RDEPEND="
+	>=x11-libs/cairo-1.10.0
+	>=dev-libs/glib-2.28.0:2
+	>=x11-libs/gtk+-3.0.0:3
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+BDEPEND="
+	${RDEPEND}
```
Comment (thesamesam) [inline x11-libs/goocanvas/goocanvas-3.0.0.ebuild:25]: You've dropped DEPEND="${RDEPEND}".

**Pull request closed without merging**